scripts/test/mender-convert-qemu

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+if [ -z "$DISK_IMG" ]; then
+    echo "ERROR: DISK_IMG is not set"
+    exit 1
+fi
+
+# Try common locations for OVMF
+ovmf_file=""
+for maybe_ovmf_file in /usr/share/OVMF/OVMF_CODE.fd /usr/share/ovmf/x64/OVMF_CODE.fd; do
+    if [ -f $maybe_ovmf_file ]; then
+        ovmf_file=$maybe_ovmf_file
+    fi
+done
+
+if [ -z "${ovmf_file}" ]; then
+    echo "ERROR: could not find OVMF file"
+    exit 1
+fi
+
+qemu-system-x86_64 \
+    -enable-kvm \
+    -nographic \
+    -m 256 \
+    -net user,hostfwd=tcp::8822-:22 \
+    -net nic,macaddr=52:54:00$(od -txC -An -N3 /dev/urandom|tr \  :) \
+    -bios ${ovmf_file} \
+    -drive format=raw,file=${DISK_IMG} &
```

Comment:
do we need some output redirection here? (as in: do we need to preserve something qemu will output to the stdout?)

Reply:
Yes, we want all the output in the terminal. This is important when tests fail to be able to inspect the GRUB boot output.